packages/protocol/test/stability/reserve.ts

```diff
@@ -0,0 +1,839 @@
+import { CeloContractName } from '@celo/protocol/lib/registry-utils'
```

Comment:
was this file deleted on master? it's not part of ASv2/ODIS work, wondering if this had to do with @bowd's work moving unit tests -> foundry? 🤔